### PR TITLE
Added README.md. Feel free to use!

### PR DESCRIPTION
**Pull request by jo-nas**
This change is [Reviewable](https://reviewable.io/reviews/google/openhtf/654)

### CODE REVIEW

**Comment by jo-nas** [comment]
Hi does someone have a look?

File: openhtf/plugs/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Usefull thinks!
+## Template to create Plugs
+* (Plug Template)[https://github.com/jo-nas/openhtf_plug_template]: A cookiecutter template for OpenHTF plugs.
+
+# List of Plugins
+* (OpenOCD Plug)[https://github.com/jo-nas/openocd_plug]: This Plug can handle commands from OpenOCD.
+* (Denkovi Relais Plug)[https://github.com/jo-nas/denkovi_plug]: A package for openhtf plugs to control the denkovi relay boards.
+* (Visa Plug)[https://github.com/jo-nas/visa_plug]: This Package contains some basic Visa Device functionality.
+* (Hameg Plugs for Openhtf)[https://github.com/jo-nas/hameg_plug]: This package contains some Hameg Plugs for the OpenHTF Framework.
+* (OpenOCD Plug)[https://github.com/jo-nas/openocd_plug]: This Plug can handle commands from OpenOCD.
```
Comment (fahhem) [inline openhtf/plugs/README.md]: repeated

File: openhtf/plugs/README.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# Usefull thinks!
+## Template to create Plugs
+* (Plug Template)[https://github.com/jo-nas/openhtf_plug_template]: A cookiecutter template for OpenHTF plugs.
```
Comment (fahhem) [inline openhtf/plugs/README.md]: I'm not sure we want to advertise this format for plugs? A single plug per repo might work for some build systems, but it's a pain in others. You're welcome to advertise it in the repos that use it though (like your 4 plugs below)!

File: openhtf/plugs/README.md [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+
+# List of Plugins
```
Comment (jettisonjoe) [inline openhtf/plugs/README.md]: s/Plugins/Plugs

I would re-word this as `List of Known Third-Party Plugs` for greater specificity.

File: openhtf/plugs/README.md (hunk 2 above)
Comment (jettisonjoe) [reply openhtf/plugs/README.md]: Agreed we shouldn't advertise/recommend this as the preferred format for plugs. Further, the cookiecutter template also does not appear to contain much if any actual plug code. Aside from a few mentions of the word "plug" in names and docstrings, this template could be used to create almost any new Python package repo. I like the list of plugs below, but let's leave the template out of this doc.

File: openhtf/plugs/README.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+# Usefull thinks!
```
Comment (jettisonjoe) [inline openhtf/plugs/README.md]: Not sure what was meant here, but let's take it out or change it to something more descriptive.

**Comment by dbhatman** [comment]
Closing this out as it was handled in https://github.com/google/openhtf/pull/768. Thanks for starting this leading to the other PR!

**Pull request closed without merging**